content/en/index.md

```diff
@@ -200,7 +200,7 @@ export default class Stars extends Component {
     }
 }
 // --repl-after
-render(&lt;Stars /&gt;, document.getElementById("app"));
+render(&lt;Stars repo="preactjs/preact" /&gt;, document.getElementById("app"));
```

Comment:
Sorry, drive-by issue that I ran into whilst testing:

While our usage example on the home page included the prop, the repl-after here was missing it. This was problematic as it'd attempt to fetch `https://github.com/undefined` if the user tried to open the example in the repl. Must've gotten missed when we added the repl-before & after stuff.